app/models/custom_button.rb

```diff
@@ -96,7 +96,7 @@ def invoke(target, source = nil)
   end
 
   def publish_event(source, target, args)
-    target.kind_of?(Array) ? target.each { |t| create_event(source, t, args) } : create_event(source, target, args)
+    target.size > 1 ? target.each { |t| create_event(source, t, args) } : create_event(source, target, args)
```

Comment:
I think `target.kind_of?(Enumerable)` would be more accurate, right?

Reply:
Could we simplify this line to: `Array(target).each { |t| create_event(source, t, args) }`